mta/entities/analysis_results.py

```diff
@@ -57,15 +57,19 @@ def __init__(self, application, project_name):
         self.application = application
         self.project_name = project_name
 
+    def get_analysis_number(self, view, row):
+        """ Method to return only digit from row text"""
+        analysis_num = view.analysis_row(row).analysis_number.text
+        only_digits = "".join([c for c in analysis_num if c.isdigit()])
```

Comment:
I think we can use digits match regular expression here.

Reply:
Hope this is okay to use